python/nav/web/auth.py

```diff
@@ -78,3 +80,14 @@ def authenticate(username, password):
         return account
     else:
         return None
+
+
+def _handle_admin_status(ldap_user, nav_account):
```

Comment:
Nitpick: `_handle_admin_status` is a bad name for something only relevant if ldap is available. `_handle_ldap_admin_status` maybe?

Reply:
Agreed :)